test/common/testlib.py

```diff
@@ -1042,6 +1043,11 @@ def setUp(self, restrict=True):
             if self.is_devel_build():
                 self.disable_preload("packagekit", "systemd")
 
+            if self.machine.image.startswith('debian') or self.machine.image.startswith('ubuntu'):
+                self.libexecdir = '/usr/lib/cockpit'
```

Comment:
The commit message of "tests: use machine.libexecdir" is misleading, as it's not a "machine" property.

This will be specific to cockpit, but IMHO that's ok -- it's very unlikely that any other project actually uses libexecdir, as they are webpack-only.

Reply:
Oops.  Missed that, thanks!